diff --color -ru upstream/public_html/config.js modified/public_html/config.js
--- upstream/public_html/config.js    2025-03-26 19:35:10.000000000 +0100
+++ modified/public_html/config.js    2025-03-29 11:18:27.550955900 +0100
@@ -28,6 +28,9 @@
 // The google maps zoom level, 0 - 16, lower is further out
 DefaultZoomLvl   = 7;

+// Which flight tracker website to use by default
+DefaultTracker = "FlightAware";
+
 // Center marker. If dump1090 provides a receiver location,
 // that location is used and these settings are ignored.

@@ -36,6 +39,18 @@
 SiteLon     = 9.0;
 SiteName    = "My Radar Site"; // tooltip of the marker

+// The receiver's ADS-B Statistics pages on the various flight tracker websites
+myAdsbStatsSiteUrlFlightAware = "";
+myAdsbStatsSiteUrlFlightradar24 = "";
+myAdsbexchangeFeederStatusUrl = "";
+myOpenskyFeederStatusUrl = "";
+myPlanefinderFeederStatusUrl = "";
+
+myAdsbfiFeederStatusUrl = "";
+myAdsbhubFeederStatusUrl = "";
+// The local FR24 Feeder Status page
+myFR24FeederStatusUrl = "";
+
 // -- Marker settings -------------------------------------

 // These settings control the coloring of aircraft by altitude.
@@ -101,7 +116,7 @@
 DefaultSiteCirclesInterval = 50;

 // Controls page title, righthand pane when nothing is selected
-PageName = "PiAware SkyAware";
+PageName = "Dump1090 plane tracker";

 // Show country flags by ICAO addresses?
 ShowFlags = true;
diff --color -ru upstream/public_html/index.html modified/public_html/index.html
--- upstream/public_html/index.html   2025-03-26 19:35:10.000000000 +0100
+++ modified/public_html/index.html   2025-03-29 11:21:21.341088100 +0100
@@ -54,8 +54,9 @@
                <div id="header" class="infoHeading">
                        <div class="logoContainer">
                                <a href="http://flightaware.com/" target="_blank"><img src="images/fa_logo_color.png" alt="FlightAware" class="flightawareLogo" srcset="images/fa_logo_color.png 1x, images/fa_logo_color@2x.png 2x, images/fa_logo_color@3x.png 3x"></a>
-                               <img src="images/pa-sa-logo.png" alt="PiAware SkyAware" class="adsbLogo piAwareLogo" srcset="images/pa-sa-logo.png 1x, images/pa-sa-logo@2x.png 2x, images/pa-sa-logo@3x.png 3x">
+                               <img src="images/pa-sa-logo.png" alt="PiAware SkyAware" class="adsbLogo piAwareLogo" srcset="images/pa-sa-logo.png 1x, images/pa-sa-logo@2x.png 2x, images/pa-sa-logo@3x.png 3x" style="display: none;">
                                <img src="images/ff-sa-logo.png" alt="FlightFeeder SkyAware" class="adsbLogo flightfeederLogo" srcset="images/ff-sa-logo.png 1x, images/ff-sa-logo@2x.png 2x, images/ff-sa-logo@3x.png 3x" style="display: none;">
+                               <a href="http://flightradar24.com/" target="_blank"><img src="images/fr24-logo.svg" alt="Flightradar24" class="flightradar24Logo"></a>
                        </div>

                        <div class="menuContainer">
@@ -210,6 +211,15 @@
                                                        <div id="dump1090_infoblock">
                                                                <div class="infoblock_heading">
                                                                        <button id="stats_page_button" class="stats_button">Go to My ADS-B Statistics Page</button>
+                                                                       <a id="stats_page_button_fa" target="_blank"><button class="stats_button">My ADS-B Statistics on FlightAware</button></a>
+                                                                       <a id="stats_page_button_fr24" target="_blank"><button class="stats_button">My ADS-B Statistics on Flightradar24</button></a>
+                                                                       <a id="fr24_feeder_status_page" target="_blank"><button class="stats_button">My FR24 Feeder Status page</button></a>
+                                                                       <a id="adsbexchange_feeder_status_page" target="_blank"><button class="stats_button">My ADSBexchange Feeder Status page</button></a>
+                                                                       <a id="opensky_feeder_status_page" target="_blank"><button class="stats_button">My Opensky Network Feeder Status page</button></a>
+                                                                       <a id="planefinder_feeder_status_page" target="_blank"><button class="stats_button">My PlaneFinder Feeder Status page</button></a>
+
+                                                                       <a id="adsbfi_feeder_status_page" target="_blank"><button class="stats_button">My ADSB.fi Feeder Status page</button></a>
+                                                                       <a id="adsbhub_feeder_status_page" target="_blank"><button class="stats_button">My ADSBHub Feeder Status page</button></a>
                                                                        <a href="https://github.com/flightaware/dump1090" id="skyaware_version" target="_blank"></a>
                                                                </div>

@@ -491,6 +501,10 @@

                                                                <div class="infoBlockTopSection">
                                                                        <span id="selected_flightaware_link" class=""></span>
+                                                                       <span id="selected_flightradar24_link" class=""></span>
+                                                                       <span id="selected_adsbexchange_link" class=""></span>
+                                                                       <span id="selected_opensky_link" class=""></span>
+                                                                       <span id="selected_planefinder_link" class=""></span>
                                                                        <div class="infoRow removePadding">
                                                                                <div class="infoHeading infoRowFluid"><span title="The alphanumeric registration code assigned by the country in which the aircraft is registered.">Registration</span>: </div>
                                                                                <div class="infoData infoRowFluid"><span id="selected_registration">n/a</span></div>
diff --color -ru upstream/public_html/script.js modified/public_html/script.js
--- upstream/public_html/script.js    2025-03-26 19:35:10.000000000 +0100
+++ modified/public_html/script.js    2025-03-29 13:25:22.925205600 +0100
@@ -461,6 +461,55 @@
                 });
         });

+        if (myAdsbStatsSiteUrlFlightAware) {
+                $('#stats_page_button_fa').attr('href',myAdsbStatsSiteUrlFlightAware);
+        } else {
+                $('#stats_page_button_fa').css('display', 'none');
+        }
+
+        if (myAdsbStatsSiteUrlFlightradar24) {
+                $('#stats_page_button_fr24').attr('href',myAdsbStatsSiteUrlFlightradar24);
+        } else {
+                $('#stats_page_button_fr24').css('display', 'none');
+        }
+
+        if (myFR24FeederStatusUrl) {
+                $('#fr24_feeder_status_page').attr('href',myFR24FeederStatusUrl);
+        } else {
+                $('#fr24_feeder_status_page').css('display', 'none');
+        }
+
+        if (myAdsbexchangeFeederStatusUrl) {
+                $('#adsbexchange_feeder_status_page').attr('href',myAdsbexchangeFeederStatusUrl);
+        } else {
+                $('#adsbexchange_feeder_status_page').css('display', 'none');
+        }
+
+        if (myOpenskyFeederStatusUrl) {
+                $('#opensky_feeder_status_page').attr('href',myOpenskyFeederStatusUrl);
+        } else {
+                $('#opensky_feeder_status_page').css('display', 'none');
+        }
+
+        if (myPlanefinderFeederStatusUrl) {
+                $('#planefinder_feeder_status_page').attr('href',myPlanefinderFeederStatusUrl);
+        } else {
+                $('#planefinder_feeder_status_page').css('display', 'none');
+        }
+
+        if (myAdsbfiFeederStatusUrl) {
+                $('#adsbfi_feeder_status_page').attr('href',myAdsbfiFeederStatusUrl);
+        } else {
+                $('#adsbfi_feeder_status_page').css('display', 'none');
+        }
+
+        if (myAdsbhubFeederStatusUrl) {
+                $('#adsbhub_feeder_status_page').attr('href',myAdsbhubFeederStatusUrl);
+        } else {
+                $('#adsbhub_feeder_status_page').css('display', 'none');
+        }
+
+
         // Force map to redraw if sidebar container is resized - use a timer to debounce
         var mapResizeTimeout;
         $("#sidebar_container").on("resize", function() {
@@ -1350,7 +1399,11 @@
         } else {
                 $('#selected_callsign').text('n/a');
         }
-        $('#selected_flightaware_link').html(getFlightAwareModeSLink(selected.icao, selected.flight, "Visit Flight Page"));
+        $('#selected_flightaware_link').html(getFlightAwareModeSLink(selected.icao, selected.flight, "FlightAware Flight Page"));
+        $('#selected_flightradar24_link').html(getFlightradar24IdentLink(selected.flight, "Flightradar24 Flight Page"));
+        $('#selected_adsbexchange_link').html(getAdsbexchangeIdentLink(selected.icao, "Adsbexchange Flight Page"));
+        $('#selected_opensky_link').html(getOpenskyIdentLink(selected.icao, "Opensky Network Flight Page"));
+        $('#selected_planefinder_link').html(getPlanefinderIdentLink(selected.flight, "PlaneFinder Flight Page"));

         if (selected.registration !== null) {
                 $('#selected_registration').text(selected.registration);
@@ -1718,18 +1771,36 @@
                         }

                         // ICAO doesn't change
+                        var flightURL = "";
+                        var flightClassName = "";
                         if (tableplane.flight) {
-                                tableplane.tr.cells[2].innerHTML = getFlightAwareModeSLink(tableplane.icao, tableplane.flight, tableplane.flight);
-                               tableplane.tr.cells[2].className = "ident_normal";
+                                flightClassName = "ident_normal";
+                                if (DefaultTracker === "Flightradar24") {
+                                        flightURL = getFlightradar24IdentLink(tableplane.flight, tableplane.flight);
+                                } else if (DefaultTracker === "Adsbexchange") {
+                                        flightURL = getAdsbexchangeIdentLink(tableplane.icao, tableplane.flight, tableplane.flight);
+                                } else if (DefaultTracker === "Planefinder") {
+                                        flightURL = getPlanefinderIdentLink(tableplane.flight, tableplane.flight);
+                                } else if (DefaultTracker === "OpenskyNetwork") {
+                                        flightURL = getOpenskyIdentLink(tableplane.icao, tableplane.flight, tableplane.flight);
+                                } else {
+                                        flightURL = getFlightAwareModeSLink(tableplane.icao, tableplane.flight, tableplane.flight);
+                                }
                         } else if (tableplane.registration !== null) {
                                 // Show registration with special styling if ident is not present
-                               tableplane.tr.cells[2].innerHTML = getFlightAwareIdentLink(tableplane.registration, tableplane.registration);
-                               tableplane.tr.cells[2].className = "ident_fallback";
+                                if (DefaultTracker === "Flightradar24" || DefaultTracker === "Adsbexchange" || DefaultTracker === "Planefinder") {
+                                        flightURL = "";
+                                        flightClassName = "";
                         } else {
-                               tableplane.tr.cells[2].innerHTML = "";
-                               tableplane.tr.cells[2].className = "";
+                                        flightURL = getFlightAwareIdentLink(tableplane.registration, tableplane.registration);
+                                        flightClassName = "ident_fallback";
                        }

+                        } else {
+                                flightURL = "";
+                                flightClassName = "";
+                        }
+                        tableplane.tr.cells[2].innerHTML = flightURL;
                         tableplane.tr.cells[3].textContent = (tableplane.registration !== null ? tableplane.registration : "");
                         tableplane.tr.cells[4].textContent = (tableplane.icaotype !== null ? tableplane.icaotype : "");
                         tableplane.tr.cells[5].textContent = (tableplane.squawk !== null ? tableplane.squawk : "");
@@ -2464,6 +2535,51 @@
     return "";
 }

+function getAdsbexchangeIdentLink(ident, linkText) {
+    if (ident !== null && ident.length > 0 && ident[0] !== '~' && ident !== "000000") {
+        if (!linkText) {
+            linkText = ident.toUpperCase();
+        }
+        return "<a target=\"_blank\" href=\"https://globe.adsbexchange.com/?icao=" + ident.trim() + "\">" + linkText + "</a>";
+    }
+
+    return "";
+}
+
+function getPlanefinderIdentLink(ident, linkText) {
+    if (ident !== null && ident !== "") {
+        if (!linkText) {
+            linkText = ident.toUpperCase();
+        }
+        return "<a target=\"_blank\" href=\"https://planefinder.net/flight/" + ident.trim() + "\">" + linkText + "</a>";
+    }
+
+    return "";
+}
+
+function getOpenskyIdentLink(ident, linkText) {
+    if (ident !== null && ident.length > 0 && ident[0] !== '~' && ident !== "000000") {
+        if (!linkText) {
+            linkText = ident.toUpperCase();
+        }
+        return "<a target=\"_blank\" href=\"https://map.opensky-network.org/?icao=" + ident.trim() + "\">" + linkText + "</a>";
+    }
+
+    return "";
+}
+
+function getFlightradar24IdentLink(ident, linkText) {
+    if (ident !== null && ident !== "") {
+        if (!linkText) {
+            linkText = ident.toUpperCase();
+        }
+        return "<a target=\"_blank\" href=\"https://flightradar24.com/" + ident.trim() + "\">" + linkText + "</a>";
+    }
+
+    return "";
+}
+
+
 function getFlightAwareModeSLink(code, ident, linkText) {
     if (code !== null && code.length > 0 && code[0] !== '~' && code !== "000000") {
         if (!linkText) {
@@ -2555,6 +2671,8 @@

 // updates the page to replace piaware with flightfeeder references
 function updatePiAwareOrFlightFeeder() {
+// PiAware and FlightFeeder logos are not displayed when running fr24feed-piaware
+/*
     if (isFlightFeeder) {
         $('.piAwareLogo').hide();
         $('.flightfeederLogo').show();
@@ -2564,6 +2682,7 @@
         $('.piAwareLogo').show();
         PageName = 'PiAware SkyAware';
     }
+*/
     refreshPageTitle();
 }

diff --color -ru upstream/public_html/style.css modified/public_html/style.css
--- upstream/public_html/style.css    2025-03-26 19:35:10.000000000 +0100
+++ modified/public_html/style.css    2025-03-29 13:16:22.390955900 +0100
@@ -567,6 +567,16 @@
     box-shadow: inset -12px 0 12px -12px #000000;
 }

+.flightradar24Logo {
+    height: 50%;
+    padding-left: 25px;
+    padding-right: 25px;
+}
+
+.logoContainer > a {
+    vertical-align: middle;
+}
+
 .adsbLogo {
     padding-left: 20px;
     padding-top: 30px;
@@ -1083,7 +1093,7 @@
     background-color: #eee;
 }

-#selected_flightaware_link a {
+#selected_flightaware_link a, #selected_flightradar24_link a, #selected_adsbexchange_link a, #selected_planefinder_link a, #selected_opensky_link a {
     background-color: #00a0e2;
     color: white;
     text-decoration: none;